src/main/java/com/smatechnologies/opcon/restapiclient/model/User.java

```diff
@@ -221,7 +223,15 @@ public void setIsSelfServiceUser(Boolean isSelfServiceUser) {
         this.isSelfServiceUser = isSelfServiceUser;
     }
 
-    @JsonIgnore
```

Comment:
File format is invalid:
- Please format file (tab are prohibed and must be replaced by spaces)